### ISSUE TITLE
Documentation: please add troubleshooting/common issues tip for SVG images

### ISSUE THREAD

**Issue by igittigitigit** [grokability/snipe-it#8735]
_**Server:**_
independent of Snipe-IT and underlying software versions

_**Is your feature request related to a problem? Please describe.**_
SVG upload is not excepting the selected file

_**Describe the solution you'd like**_
I've seen users dealing with the same problem before and after I had my own ticket here. There was never a solution given. For myself I found the reason of this issue after my ticket was already closed and I would like to have this troubleshooting tip added to the official documentation. Of course, I am not sure if really everyone with this issue had the same cause, but it looks like it.

If the upload of a SVG files fails, make sure that the following two lines are include on top of the SVG file, **before** the `<svg` section starts.

```
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
```
With this correct header information an upload works fine.

**Comment by snipe** [comment]
If it's not a valid SVG, it wouldn't upload correctly. I'm not sure how we'd explain that in a way that wouldn't be more confusing for the average user (who usually isn't responsible for the creation of the image, just for uploading it.)

**Comment by snipe** [comment]
I've added it to the documentation, although I'm not sure it won't cause as much confusion as it solves problems. 

https://snipe-it.readme.io/docs/common-issues#svg-images-will-not-upload

<blockquote><img src="/favicon.ico" width="48" align="right"><div>Snipe-IT Documentation</div><div><strong><a href="https://snipe-it.readme.io/docs/common-issues#svg-images-will-not-upload">Common Issues</a></strong></div><div>  These commands should be run from your Snipe-IT project root, just like every other cli command we mention. They should not need to be run as root/sudo/admin.These commands clear out cached service files that help speed up modern PHP applications but can also very occasionally be out of date, resu...</div></blockquote>

**Comment by igittigitigit** [comment]
Thanks for adding it!

I'm not an average user, working in the IT for a long time by now. But I just downloaded SVG files of manufacture logos (Dell, Microsoft, etc.) from the Internet/Wikipedia. All were working fine with many programs, like viewing them with Internet Explorer. Therefore I wasn't thinking of an non conform XML content. Only after I tried with a different file, which was working, I compared this file with the non working ones and I figured it out. That means, that other programs just writing and interpreting <svg> content without adding or checking a valid header. (Which isn't good, but unfortunately confusing if you don't know it.)

I'm not sure if an average user will even check the github pages for this error/info message. If its happening, it will go to the person who has installed Snipe-IT. In our case, it was me.

Long text short, it would have helped me and I hope it helps others. So, thanks again for adding it. And thanks a lot for this great asset management solution!
